packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.spec.tsx

```diff
@@ -6,8 +6,7 @@ import HorizontalOverflow, {
     HorizontalOverflowClassNameContract,
 } from "./";
 import "raf/polyfill";
-import { ConstructableResizeObserver } from "./resize-observer";
-import { DisplayNamePrefix } from "../utilities";
+import { ConstructableResizeObserver, DisplayNamePrefix } from "../utilities";
```

Comment:
Your merge message uses "Constructible", this is spelled "Constructable" I believe -ible to be more correct because Oxford dictionary does not specify the other spelling. Either way we need it to be consistent.

Reply:
"ible" it is